Rubberduck.VBEditor.VBA/SafeComWrappers/Application/AccessApp.cs

```diff
@@ -84,6 +96,71 @@ public override IEnumerable<HostDocument> GetDocuments()
             return list;
         }
 
+        public override bool CanOpenDocumentDesigner(QualifiedModuleName moduleName)
+        {
+            return TryGetDocument(moduleName, out _);
```

Comment:
Do we know if this is side-effecting?

Reply:
No. It will ultimately get a `HostDocument` struct which is discarded. This suffices for the purpose of proving that such document exists.

Reply:
I mean side effecting in the VBE environment (like popping a window into view or something like that). I was mostly curious if it needed a comment.

Reply:
Yes, I figured that's what you meant and it's still no; this particular path through Access' OM should not have any effects in VBE.